### ISSUE TITLE
Calibration button

### ISSUE THREAD

**Issue by Gabriele69** [sverzegnassi/ubuntu-touch-level-app#1]
It would be useful to have a button that allows zeroing on the horizontal and / or vertical plane to correct any inaccuracies with a certain planar reference

**Comment by sverzegnassi** [comment]
Agreed, thanks for reporting it! Adding `help wanted` and `good first issue` labels to the report.
I'm going afk for a few days, so I can't provide the patch by myself.

I don't expect it to be particularly complex; adding a QML [Settings component](http://doc.qt.io/qt-5/qml-qt-labs-settings-settings.html) as backend should do the work.

Patches welcome!

**Comment by Gabriele69** [comment]
Hi Stefano.
Have you abandoned Level's development?

g
